linkatos/firebase.py

```diff
@@ -0,0 +1,33 @@
+import pyrebase
+
+
+def store_url(is_yes, url, FB_API_KEY, FB_USER, FB_PASS):
+    # connect to firebase
+    config = {
+        "apiKey": FB_API_KEY,
+        "authDomain": "coses-acbe6.firebaseapp.com",
+        "databaseURL": "https://coses-acbe6.firebaseio.com",
+        "storageBucket": "coses-acbe6.appspot.com"}
+
+    # do nothing if it's unnecessary
+    if is_yes is False:
+        return is_yes
+
+    # otherwise keep url
+    firebase = pyrebase.initialize_app(config)
```

Comment:
So, every time you call `store_url()` you set up firebase config. Is this intentional?